### DISCUSSION TITLE
Any way to have "Inter Display" in font variable ?

### DISCUSSION THREAD

**Discussion by hexionaut** [rsms/inter#757] [Q&A] [answered]
Hi,

I think everything is in the title.

But from what I know, it's not possible, right ?

**Comment by kenmcd** [comment, marked as answer]
Yes, it is possible.
The Optical Size (`opsz`) axis set to 32pts is the same as the static "Display" fonts.

EDIT: I just realized the the variable fonts do not have named instances equivalent to the static "Display" fonts.
Is that what you mean?
What application are you using?
Does that application only support pre-made named instances (such as LibreOffice, Word), or does it support full variable axis settings?

Inter really needs to have named instances for the Display optical size at a minimum, maybe more.
Going to see if there is an easy way add them without breaking anything else.

**Reply by hexionaut** [reply]
@kenmcd I would like to be able on the same webpage to use Inter for paragraph text and Inter Display as the font for the headings.

That's why I was looking for a way to use the same font for both needs.

**Reply by intersk** [reply]
use CSS and property Font Family

**Reply by hexionaut** [reply]
I am using tailwind so this is what I did :

```
  fontFamily: {
    sans: ['Inter Variable', ...defaultTheme.fontFamily.sans],
    heading: ['Inter Variable', ...defaultTheme.fontFamily.sans, {
      fontVariationSettings: "'opsz' 32",
    }],
  }
```

However, cannot see any difference between font using sans (bottom) and heading (top).

![t6o7LLKVtEdDAW0x](https://github.com/user-attachments/assets/d464e156-5ba1-4d6b-a653-709fdf5911c4)

**Reply by intersk** [reply]
```
@font-face {
    font-family: "Inter";
    src: url("Inter.woff2") format("woff2"),
         url("Inter.woff") format("woff"),
         url("Inter.ttf") format("truetype");
}
@font-face {
    font-family: "Inter Display";
    src: url("Inter Display.woff2") format("woff2"),
         url("Inter Display.woff") format("woff"),
         url("Inter Display.ttf") format("truetype");
}

p { 
  font-family: "Inter";
}
span {
  font-family: "Inter Display";
}

```
